lib/src/commonMain/kotlin/app/moviebase/trakt/api/TraktShowsApi.kt

```diff
@@ -32,6 +32,16 @@ class TraktShowsApi(private val client: HttpClient) {
         extended?.let { parameterExtended(it) }
     }
 
+    suspend fun getAnticipated(
+        page: Int,
+        limit: Int,
+        extended: TraktExtended? = null,
+    ): List<TraktAnticipatedShow> = client.getByPaths(*pathShows("anticipated")) {
```

Comment:
@chrisbanes The import is missing in this class:
`import app.moviebase.trakt.model.TraktAnticipatedShow`

Reply:
woops, fixed